code/game/objects/structures/maintenance.dm

```diff
@@ -47,7 +47,7 @@ at the cost of risking a vicious bite.**/
 	if(!isliving(user))
 		return FALSE
 	var/mob/living/living_user = user
-	if((living_user.mobility_flags & MOBILITY_STAND) && ishuman(living_user)) //I dont think monkeys can crawl on command.
+	if((living_user.mobility_flags & MOBILITY_STAND) || !ishuman(living_user)) //I dont think monkeys can crawl on command.
 		return FALSE
 	return TRUE
```

Comment:
by the way, while you're here, can you fix the attackby() proc for moisture traps so that janiborgs can fill their buckets from them?